src/stan_math_backend/Expression_gen.ml

```diff
@@ -223,75 +216,68 @@ and pp_scalar_binary ppf scalar_fmt generic_fmt es =
 
 and gen_operator_app = function
   | Operator.Plus ->
-      fun ppf es -> pp_scalar_binary ppf "(%a@ +@ %a)" "add(@,%a,@ %a)" es
+      fun ppf es ->
+        pp_scalar_binary ppf "(%a@ +@ %a)" "stan::math::add(@,%a,@ %a)" es
```

Comment:
I am not sure this special casing is needed at all anymore, as we also have a definition of add for scalars: https://github.com/stan-dev/math/blob/develop/stan/math/prim/fun/add.hpp

The same goes for all operators I think. If any of them are missing its probably just oversight rather than intention.

Reply:
I think you are right, with a few exceptions like transpose which don't seem to have overloads. That said, it ends up being very much worth it IMO, to avoid things like

` pos - 1` turning into `stan::math::minus(pos,-1)` in the generated code. I've cleaned up this code a little though

Reply:
Oh yeah, that would be verbose and ugly. Good call.